projects/crates.io/stego/package.yml

```diff
@@ -0,0 +1,22 @@
+distributable:
+  url: https://github.com/ajmwagar/stego/archive/d9d5911f4d2d141fea74936f235a74bf03961c71.zip
+  strip-components: 1
+
+provides:
+  - bin/stego
+
+versions:
+  - 0.0.0
```

Comment:
Current convention is that this be the calver of the commit date (2023.08.25, for example).